streaming/java/BUILD.bazel

```diff
@@ -43,12 +45,44 @@ define_java_module(
     ],
     visibility = ["//visibility:public"],
     deps = [
+        ":org_ray_ray_streaming-state",
         "@ray_streaming_maven//:com_google_guava_guava",
         "@ray_streaming_maven//:org_slf4j_slf4j_api",
         "@ray_streaming_maven//:org_slf4j_slf4j_log4j12",
     ],
 )
 
+define_java_module(
+    name = "streaming-state",
+    define_test_lib = True,
+    test_deps = [
+        "@ray_streaming_maven//:com_google_guava_guava",
+        "@ray_streaming_maven//:org_slf4j_slf4j_api",
+        "@ray_streaming_maven//:org_slf4j_slf4j_log4j12",
+        "@ray_streaming_maven//:org_testng_testng",
+        "@ray_streaming_maven//:org_mockito_mockito_all",
+        "@ray_streaming_maven//:com_esotericsoftware_kryo",
+        "@ray_streaming_maven//:com_esotericsoftware_minlog",
+        "@ray_streaming_maven//:com_esotericsoftware_reflectasm",
+        "@ray_streaming_maven//:de_javakaffee_kryo_serializers",
```

Comment:
Can we use `fst` for serialization? streaming java jars is packaged into ray python wheel. We should avoid dependencies if not very necessary. And It seems that  `fst` has a better performance than `kryo`.

Reply:
ok